packages/delayedInbox-l2msg/.env-sample

```diff
@@ -5,7 +5,7 @@
 DEVNET_PRIVKEY ='0x your key here'
 
 # Hosted Aggregator Node (JSON-RPC Endpoint)
-L2RPC="https://rinkeby.arbitrum.io/rpc"
+L2RPC="https://goerli-rollup.arbitrum.io/rpc"
 
 # Ethereum RPC i.e., for rinkeby https://rinkeby.infura.io/v3/<your infura key>
```

Comment:
We could update this with The RPC URL for the Goerli as well:

`Ethereum RPC; i.e., for Goerli https://goerli.infura.io/v3/<your infura key>`

Reply:
Also, if we decide to have users to add the Inbox address themselves, we need to add it to the env sample as well.